cmake/XcodeHelpers.cmake

```
@@ -0,0 +1,29 @@
+# Inspired by https://catfox.life/2022/08/14/compiling-xibs-with-cmake-without-xcode/
+function(target_xib_resources TARGET)
+    cmake_parse_arguments(PARSE_ARGV 1 _pxib
+        ""
+        ""
+        "SOURCES"
+    )
+
+    # Xcode will automatically run ibtool to compile the XIB files into NIB
+    # resources, but if we're generating Makefiles or Ninja projects then we
+    # need to handle that ourselves...
+    if (CMAKE_GENERATOR STREQUAL "Xcode")
+        target_sources(${TARGET} PRIVATE ${_pxib_SOURCES})
+    else()
+        find_program(IBTOOL ibtool HINTS "/usr/bin" "${OSX_DEVELOPER_ROOT}/usr/bin")
+        if (${IBTOOL} STREQUAL "IBTOOL-NOTFOUND")
```

```suggestion
        if (NOT IBTOOL)
```

or use the `REQUIRED` argument on the `find_program` call